mobot/mobot_client/item_drop_session.py

```diff
@@ -0,0 +1,563 @@
+# Copyright (c) 2021 MobileCoin. All rights reserved.
+
+import os
+import full_service_cli as mc
+import googlemaps
+import enum
+import pytz
+
+from mobot_client.drop_session import BaseDropSession, ItemSessionState
+from mobot_client.models import (
+    DropSession,
+    CustomerStorePreferences,
+    Order,
+    Sku,
+    CustomerDropRefunds,
+)
+from mobot_client.chat_strings import ChatStrings
+
+
+class OrderStatus(enum.Enum):
+    STARTED = 0
+    CONFIRMED = 1
+    SHIPPED = 2
+    CANCELLED = 3
+
+
+class ItemDropSession(BaseDropSession):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        gmaps_client_key = os.environ["GMAPS_CLIENT_KEY"]
+        self.gmaps = googlemaps.Client(key=gmaps_client_key)
+        
+        self.vat_id = os.environ["VAT_ID"]
+
+    @staticmethod
+    def drop_item_has_stock_remaining(drop):
+        # FIXME: Unused
+        skus = Sku.objects.fliter(item=drop.item)
```

Comment:
Spelling: `filter`. But we can do a ForeignKey instead and pull the skus off the drop.item.skus field.

Reply:
@sugargoat This is fixed in my clean-up branch.